playbooks/roles/deploy-osh/defaults/main.yml

```diff
@@ -96,4 +96,11 @@ suse_osh_deploy_memcached_yaml_overrides: {}
 suse_osh_deploy_nova_yaml_overrides: {}
 suse_osh_deploy_ovs_yaml_overrides: {}
 suse_osh_deploy_rabbitmq_yaml_overrides: {}
-suse_osh_deploy_neutron_yaml_overrides: {}
\ No newline at end of file
+
+suse_osh_deploy_neutron_yaml_overrides:
+  # api-paste entrypoint neutron.api.versions:Versions.factory was deprecated in Queens
+  # We must replace it for Rocky deployments
+  conf:
+    paste:
+      app:neutronversions:
```

Comment:
shouldn't this need to be wrapped in double quotes?

Reply:
I don't know ... it's the same value as in `values.yaml`

Reply:
Semi Complex keys in yaml should be wrapped with "". And yes, "" is removed when parsing anyway, but it's for clarity of the key:value, here the key contains ":"

But ok, if it's like that in values, let's keep it.

Reply:
works for me like that, tested. I was also surprised :o

Reply:
Can we make this part of the template instead? I don't see a reason we should not provide this by default in the template nowadays.

Reply:
True